internal/aez/ccd_data.go

```diff
@@ -155,21 +188,92 @@ type CCDImagePackData struct {
 // Read CCDImagePackData from the buffer
 //
 // returns the BC (bad columns) array and the error status.
-func (ccdImagePackData *CCDImagePackData) Read(buf io.Reader) ([]uint16, error) {
-	err := binary.Read(buf, binary.LittleEndian, ccdImagePackData)
+func (ccd *CCDImagePackData) Read(buf io.Reader) ([]uint16, error) {
+	err := binary.Read(buf, binary.LittleEndian, ccd)
 	if err != nil {
 		return nil, err
 	}
-	badColumns := make([]uint16, ccdImagePackData.NBC)
+	badColumns := make([]uint16, ccd.NBC)
 	return badColumns, binary.Read(buf, binary.LittleEndian, &badColumns)
 }
 
 // Time returns the measurement time in UTC
-func (ccdImagePackData *CCDImagePackData) Time(epoch time.Time) time.Time {
-	return ccsds.UnsegmentedTimeDate(ccdImagePackData.EXPTS, ccdImagePackData.EXPTSS, epoch)
+func (ccd *CCDImagePackData) Time(epoch time.Time) time.Time {
+	return ccsds.UnsegmentedTimeDate(ccd.EXPTS, ccd.EXPTSS, epoch)
 }
 
 // Nanoseconds returns the measurement time in nanoseconds since epoch
-func (ccdImagePackData *CCDImagePackData) Nanoseconds() int64 {
-	return ccsds.UnsegmentedTimeNanoseconds(ccdImagePackData.EXPTS, ccdImagePackData.EXPTSS)
+func (ccd *CCDImagePackData) Nanoseconds() int64 {
+	return ccsds.UnsegmentedTimeNanoseconds(ccd.EXPTS, ccd.EXPTSS)
+}
+
+// CSVHeaders returns the exportable field names
+func (ccd CCDImagePackData) CSVHeaders() []string {
+	return []string{
+		"CCDSEL",
+		"EXP Nanoseconds",
+		"EXP Date",
+		"WDW Mode",
+		"WDW InputDataWindow",
+		"WDWOV",
+		"JPEGQ",
+		"FRAME",
+		"NROW",
+		"NRBIN",
+		"NRSKIP",
+		"NCOL",
+		"NCBIN FPGAColumns",
+		"NCBIN CCDColumns",
+		"NCSKIP",
+		"NFLUSH",
+		"TEXPMS",
+		"GAIN Mode",
+		"GAIN Timing",
+		"TEMP",
+		"FBINOV",
+		"LBLNK",
+		"TBLNK",
+		"ZERO",
+		"TIMING1",
+		"TIMING2",
+		"VERSION",
+		"TIMING3",
+		"NBC",
+	}
+}
+
+// CSVRow returns the exportable field values
+func (ccd CCDImagePackData) CSVRow() []string {
+	wdwhigh, wdwlow, _ := ccd.WDW.InputDataWindow()
+	return []string{
+		strconv.Itoa(int(ccd.CCDSEL)),

```

Comment:
I'd prefer `fmt.sprintf("%i", ccd.CCDSL)` type of conversions...

Reply:
I think `strconv` is preferred idiomatic way to convert ints and floats to strings and should be faster too.